package.json

```diff
@@ -177,6 +177,7 @@
     "husky": "^9.0.11",
     "lint-staged": "15.4.3",
     "nock": "13.5.6",
+    "node-gyp": "^11.2.0",
```

Comment:
Why did you add this dependency? We don't need it. Please remove this line and the updates to yarn.lock.